random_menu.rb

```diff
@@ -0,0 +1,20 @@
+require "colorize"
+
+first = ["Purple".magenta, "Yummy".light_blue.underline, "Silky".red.underline, "Orange".light_red,
+  "Mushy".green, "Crunchy".light_yellow.bold, "Fizzy".blue, "Gooey".cyan.bold, "Slimy".light_green,
+  "Dry".yellow, "Warm".light_magenta.bold, "Moist".light_cyan.bold, "Hot".light_white]
+second = ["Baked".red.underline, "Fried".light_magenta.bold, "Steamed".light_cyan, "Creamed".light_yellow,
+  "Boiled".blue, "Sauteed".yellow.bold, "Preserved".cyan.underline, "Grilled".magenta, "Frozen".light_blue.underline,
+  "Raw".light_red.bold, "Fermented".green,"Rolled".light_green.bold, "Stacked".light_white.underline]
```

Comment:
Just watch your use of whitespace after the comma - I think you got it in almost all places but missed in some